templates/base/Dockerfile

```diff
@@ -3,10 +3,33 @@ FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes build-essential ca-certificates jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip wget && \
 	apt-get upgrade --yes && \
 	apt-get clean
 
+RUN VERSION=8.7.1 && \
+	wget https://curl.haxx.se/download/curl-${VERSION}.tar.gz && \
+	tar -xzvf curl-${VERSION}.tar.gz && \
+	rm -f curl-${VERSION}.tar.gz && \
+	cd curl-${VERSION} && \
+	./configure --prefix=/usr/local --with-ssl --with-nghttp2 && \
```

Comment:
@Wykenz, can you please confirm if the `--prefix=/usr/local` is indeed the default place of `curl` installation?

I mean, if this new version is installed in the same place of the previous `curl` version, the one having the security vulnerability.